contracts/RelaySugar.vy

```diff
@@ -54,61 +64,154 @@ interface IAutoCompounder:
 
 # Vars
 factory: public(IAutoCompounderFactory)
-ve_sugar: public(IVeSugar)
+voter: public(IVoter)
 ve: public(IVotingEscrow)
+token: public(address)
 
 @external
-def __init__(_factory: address, _ve_sugar: address, _ve: address):
+def __init__(_factory: address, _voter: address):
   """
   @dev Set up our external factory contract
   """
   self.factory = IAutoCompounderFactory(_factory)
-  self.ve_sugar = IVeSugar(_ve_sugar)
-  self.ve = IVotingEscrow(_ve)
+  self.voter = IVoter(_voter)
+  self.ve = IVotingEscrow(self.voter.ve())
+  self.token = self.ve.token()
 
 @external
 @view
-def all() -> DynArray[Relay, MAX_COMPOUNDERS]:
+def all(_account: address) -> (DynArray[Relay, MAX_COMPOUNDERS], DynArray[VeNFT, MAX_RESULTS]):
+  """
+  @notice Returns all AutoCompounders and account's deposits
```

Comment:
If we no longer need the `deposits` I suggest we update the docstring to indicate that a `Relay` struct is a mix of the (m)veNFT data and the compounder data :pray: